README.md

```diff
@@ -11,8 +11,9 @@ Both Social Team members and Node.js project collaborators submitting PRs should
   - Hashtags like #nodejs or #javascript are relatively unused outside of spam at this point, and add no additional semantic value on Twitter.
 - We do not amplify or share content that could unduly promote or benefit one corporation that does work in the space of Node.js over another.
 - We try to amplify the collaborators who are doing the work rather than anonymously taking credit for it as the project.
-  - For general releases, we try to amplify those on the Release Team who are cutting releases directly rather than tweeting release announcements directly from the project's account.
-  - For security releases, it is important that the tweet come directly from the Node.js Twitter account (the authoritative source) so that readers will immediately recognize the announcement. Additionally, if there is an announcement tweet from collaborators who put in the effort to make the security release happen, those tweets should also be amplified with a retweet from the Node.js account.
+  - For minor and patch releases, we try to amplify those on the Release Team who are cutting releases directly rather than tweeting release announcements directly from the project's account.
+    - For major releases, it is important that the tweet come directly from the Node.js Twitter account (the authoritative source) so that readers will immediately recognize the announcement. Direct tweets for majors helps ensure maximal engagement and reach of the project's release. Additionally, if there is an announcement tweet from collaborators who put in the effort to make the security release happen, those tweets should also be amplified with a retweet from the Node.js account.
```

Comment:
I'm not sure I agree that the announcement of major releases are `time sensitive`. I'd suggest that we amplify the tweet from those doing the release first and then later follow up with a tweet from the Node.js twitter account. I know I've been frusted in the past when tweets have gone out and the release is not even fully complete yet. Gating on the tweets from those doing the release work would make sure we avoid that.

Reply:
> I'm not sure I agree that the announcement of major releases are time sensitive.

Where does this mention time sensitive? I'm confused, that's not something that's asserted anywhere.

> I'd suggest that we amplify the tweet from those doing the release first and then later follow up with a tweet from the Node.js twitter account.

I'm 100% fine doing this simultaneously or near-simultaneously. The challenge with this is that because of how Twitter works, first-party tweets get more algorithmic boost over retweets _and_ people are much more likely to directly engage with a tweet from the project. With something like a major announcement, having an initial or simultaneous first-party tweet will ensure that more of our users are alerted to the fact that there's a new major.

> I know I've been frusted in the past when tweets have gone out and the release is not even fully complete yet. Gating on the tweets from those doing the release work would make sure we avoid that.

Further, the current process in-practice _should never_ have a tweet from @nodejs go out before the release happens, since we wait for the release team ping us, so I'm not sure where that concern is coming from. If you can point to a recent example of that, I'm more than happy to dig in and figure out what went wrong and how we can learn from it. Not trying to invalidate your feelings of frustration, but I'm not sure what's being talked about here and I want to make sure we're all on the same page - it makes things challenging if we're not.

Reply:
(Also, apologies for having this discussion get marked as outdated - I changed some incorrect spacing and that hid the discussion. Not my intention, still want to see this resolved - just an artifact of GitHub UI.)